src/FBGraphRequestManager.ts

```diff
@@ -49,12 +47,14 @@ function _verifyParameters(request: GraphRequest) {
 
 class FBGraphRequestManager {
   requestBatch: Array<GraphRequest>;
-  requestCallbacks: Array<?Callback>;
+  requestCallbacks: Array<Callback | undefined>;
   batchCallback: Callback;
 
   constructor() {
     this.requestBatch = [];
     this.requestCallbacks = [];
+    // TODO: [TS Migration]: Added default value to fix type definitions. Discuss before merging.
+    this.batchCallback = () => {};
```

Comment:
Isn't the case here to just use `this.batchCallback?.()` and then you won't need to put a default value here?
But either way having a default noop here seems fine to me.